droidlet/tools/artifact_scripts/upload_artifacts_to_aws.py

```diff
@@ -14,45 +14,32 @@
 print("Rootdir : %r" % ROOTDIR)
 
 
-def compute_checksum_tar_and_upload(agent, artifact_name, model_name=None):
+def tar_and_upload(agent, artifact_name, model_name=None, checksum=None):
+    """
+    Tar all files for the artifact and upload it to AWS.
+    """
     if not agent:
```

Comment:
I'm not sure why we need to do this step all over again ? `compute_checksum` already does this for us. We can just start from :
```
print("Now making the tar file...")
process = Popen(...
```
right ?

Reply:
My point is just give these as input to the function rather than recomputing artifact_path_name